.github/workflows/CI.yml

```diff
@@ -737,7 +737,7 @@ jobs:
 
   build_win:
     name: Windows
-    runs-on: windows-2019
+    runs-on: windows-2022
```

Comment:
Is this change necessary for your PR? I'd prefer if this change were in a separate PR.